Buildings/Fluid/SolarCollectors/Data/BaseClasses/Generic.mo

```diff
@@ -0,0 +1,76 @@
+within Buildings.Fluid.SolarCollectors.Data.BaseClasses;
+record Generic
+  "Generic data record providing common inputs for ASHRAE93 and EN12975 solar collector models"
+  extends Modelica.Icons.Record;
+
+  parameter Modelica.Units.SI.Area A "Area";
+  parameter Buildings.Fluid.SolarCollectors.Types.HeatCapacity CTyp
+    "Total thermal capacity or fluid volume and 'dry' thermal capacity or mass";
+  parameter Modelica.Units.SI.HeatCapacity C
+    "Dry or total thermal capacity of the solar thermal collector";
+  parameter Modelica.Units.SI.Volume V "Fluid volume ";
+  parameter Modelica.Units.SI.Mass mDry "Dry mass";
+  parameter Real mperA_flow_nominal(unit="kg/(s.m2)")
+    "Nominal mass flow rate per unit area of collector";
+  parameter Modelica.Units.SI.PressureDifference dp_nominal(displayUnit="Pa")
+    "Nominal pressure drop";
+  parameter Real b0 "1st incident angle modifier coefficient";
+  parameter Real b1 "2nd incident angle modifier coefficient";
```

Comment:
Add missing unit attribute

Reply:
I added a `unit` attribute, as well as a `min` and `max` attribute.